forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+import android.content.Context
+import android.content.SharedPreferences
+
+interface ConfigInterface {
+    fun loadFromCache(context: Context?): FROptions
+}
+
+class ConfigHelper: ConfigInterface {
+
+    companion object {
+
+        const val ORG_FORGEROCK_V_1_HOSTS = "org.forgerock.v1.HOSTS"
+
+        @JvmStatic
+        fun cache(context: Context?, frOptions: FROptions) {
```

Comment:
Should we rename to persist()? cache seem caching in memory.

Reply:
i like persist